components/gitpod-db/src/typeorm/transformer.ts

```diff
@@ -38,6 +38,18 @@ export namespace Transformer {
         },
     };
 
+    export const MAP_ISO_STRING_TO_TIMESTAMP_DROP_NULLABLE: ValueTransformer = {
+        to(value: any): any {
+            return MAP_ISO_STRING_TO_TIMESTAMP_DROP.to(value);
+        },
+        from(value: any): any {
+            if (value == null) {
+                return null;
```

Comment:
👍

Reply:
I got three comments:
 1. `null` is the wrong value to return. Judging from the interface, [we expect](https://github.com/gitpod-io/gitpod/pull/10573/files#diff-957d9dd15b947e3969539ebd0dabcb8b40ce8b5564e5820cd9bb27d88be1dbc7R369) `undefined` here.
 2. the transformer is "asymmetric" (from returns `null`, to returns `undefined`). Which leads to weird bugs if you do write-read-write cycles. Thus please do stick to 3. :point_down: 
 3. the default pattern we use for "nullable" timestamp columns that is know to work correctly is this:
   1. string column type, NOT nullable, default to "" (empty string)
   2. use MAP_EMPTY_STR_TO_UNDEFINED ([example](https://github.com/gitpod-io/gitpod/blob/1420b9ca546bcc7bb140feb9bc201dcf24acf318/components/gitpod-db/src/typeorm/entity/db-workspace.ts#L88-L92))
 
 I know that we do it differently in other occasions. But if you ask me, because the underlying tech (mysql) is so unsatisfying, we should try hard to stick to one pattern how to work around it. and 3. is the way we recommend. :lotus_position: